syntax = "proto3";


service UserService {
   
   rpc GetUser (UserRequest) returns (UserResponse) {}
}

message UserRequest {
   string user_id = 1;
}

message UserResponse {
   string user_id = 1;
   string name = 2;
   string email = 3;
}
